src/renderer/features/governance/aggregates/details.ts

```diff
@@ -12,6 +13,8 @@ import { votingAggregate } from './voting';
 
 const flow = createGate<{ chain: Chain; referendum: Referendum }>();
 
+const $canVote = walletModel.$activeWallet.map((wallet) => !walletUtils.isWatchOnly(wallet));
```

Comment:
we have permissionUtils for such checking

Reply:
good point, tnx